distributions/openhab/src/main/resources/bin/backup.bat

```diff
@@ -0,0 +1,19 @@
+@ECHO off
+SETLOCAL
+
```

Comment:
If you're planning on users running this from the BIN folder then maybe something like this. Especially if you want them to be able to Right-click - Run as admin.

cd %~dp0
cd ..\..

Reply:
I didn't bother with that in the batch files since the powerscript files do that (that way you get the same functionality regardless if you call it from the batch or powerscript files directly).

Reply:
ah - I see what you mean by that.  Will fix